polkadot/node/network/bridge/src/metrics.rs

```diff
@@ -105,9 +105,16 @@ impl Metrics {
 
 	pub fn on_report_event(&self) {
 		if let Some(metrics) = self.0.as_ref() {
+			metrics.messages_sent.with_label_values(&["report_peer"]).inc();
```

Comment:
call `self.on_message("report_peer")` ?

Reply:
It would have double `if let Some(metrics)` but presumable it'll be free due to branch predictions locality.